docs/language.rst

```diff
@@ -177,6 +177,122 @@ Lists of primitive types are also primitive types: ``string[]``, ``number[]``, `
     # incorrect, can't assign to dict after construction
     # a["otherkey"] = "othervalue"
 
+Strings
++++++++
+
+
+There are four kinds of strings in the Inmanta language:
+
+- regular strings
+
+.. code-block:: inmanta
+
+    regular_string_1 = "This is...\n...a basic string."
+
+    # Expected output e.g. when displayed:
+    # This is...
+    # ...a basic string.
+
+
+    regular_string_2 = 'This one too.'
```

Comment:
Rendering is weird here: some red boxes around the quotes and `This` is displayed in blue
![image](https://github.com/inmanta/inmanta-core/assets/98876549/c39e8386-462c-4d80-83c0-97bd0ff5c155)